src/include/migraphx/op/common.hpp

```diff
@@ -23,6 +23,23 @@ enum class rnn_direction
     bidirectional,
 };
 
+// different attributes
+// 1) use_input(default)/use_output
+// 2) use_rank(default)/use_len
+// 3) clip_min(default)/not_clip_min
+//   3.1) include_min(default)/exclude_min
+// 4) clip_max(default)/not_clip_max
+//   4.1) exclude_max(default)/include_max
+enum class op_normalize_attributes
```

Comment:
Could we name this `normalize_attribute`?

Reply:
fixed